scipy/stats/_distn_infrastructure.py

```diff
@@ -1209,14 +1209,9 @@ def entropy(self, *args, **kwds):
         place(output, cond0, self.vecentropy(*goodargs) + log(goodscale))
         return output[()]
 
```

Comment:
Please mak these changes up here, too:
https://github.com/scipy/scipy/blob/d6b06f5ecc7f09d205b2fd2ce17c4254b4582f07/scipy/stats/_distn_infrastructure.py#L493-L500

(see how they were added in 13490 - I think `*args` and `*kwds` can be removed from the signature up there.)